flotta-agent.spec

```diff
@@ -13,7 +13,11 @@ Source0:    %{name}-%{version}.tar.gz
 
 BuildRequires:  golang
 
+%if 0%{?rhel}
```

Comment:
Should this be `{?el9}` or something like {?rhel} >= 9

Reply:
Rhel 8 also has ansible-core, I discovered afterwards.